docs/getting-started.md

```diff
@@ -0,0 +1,140 @@
+﻿
+# Kubevirtci
```

Comment:
The content of the doc is ok, but the structure is really hard to follow, especially for someone new into project.

Please reference existing docs (like setting up the dev env).

Remember that the target audience are kubevirtci contributors that want to add a new flavour of provider.

The structure should be:

1. What is kubevirtci and how is it helpful, that you already have here
2. What does the running provider consist off
3. What happens during `cluster-up`, one 
4. The structure of the project
5. Simple example of how to create a new provider (e.g. install one additional manifest on k8s), it would be helpful if it was a step by step tutorial from checking out of the code all the way to pushing a PR. It should be enough (or reference enough) for a new dev to contribute.